src/scancode/cli.py

```diff
@@ -518,7 +541,7 @@ def _scanit(paths, scanners, scans_cache_class, diag,
     return success, rel_path
 
 
-def resource_paths(base_path):
+def resource_paths(base_path, user_ignores):
```

Comment:
@pombredanne Is it alright if I modify the existing methods in the test_ignore_files.py https://github.com/yashdsaraf/scancode-toolkit/blob/develop/tests/scancode/test_ignore_files.py with asserts for this function? Or is it preferable to write separate methods?